apps/zbugs/server/email.ts

```diff
@@ -0,0 +1,67 @@
+import nodemailer from 'nodemailer';
+
+async function getTransport() {
+  if (process.env.LOOPS_EMAIL_API_KEY !== undefined) {
+    const transport = nodemailer.createTransport({
```

Comment:
Surprised to see that we're using smtp directly.

This seems very likely to be a pita. Can we just use loops' REST API.